packages/frontend/core/src/components/workspace/index.tsx

```diff
@@ -36,22 +37,26 @@ export interface MainContainerProps extends HTMLAttributes<HTMLDivElement> {
   className?: string;
   padding?: boolean;
   transparent?: boolean;
+  clientBorder?: boolean;
```

Comment:
I think clientBorder === padding in this case.

Reply:
Indeed!😝 Removed.